src/commandDetails/coin/leaderboard.ts

```diff
@@ -47,9 +48,16 @@ const getCurrentCoinLeaderboardEmbed = async (
       .join('\\_')
       .split('`')
       .join('\\`');
-    const userCoinEntryText = `${leaderboardArray.length + 1}. ${cleanUserTag} - ${
+    const userCoinEntryText = `${rank}. ${cleanUserTag} - ${
```

Comment:
The leaderboard would currently show the person in 1st place to have a rank of 0?

Reply:
So sorry. I didn't change much last time so I thought I got it correct. Just updated it and checked it.
<img width="479" alt="image" src="https://user-images.githubusercontent.com/84613574/196548470-4afce0a9-fd37-47b5-9f27-f7bc5eb86afb.png">
The ranking should be good now. @Picowchew